pkg/pages/error.go

```diff
@@ -175,15 +175,26 @@ type ErrorData struct {
 	Error string
 }
 
-func GetErrorPage(name string, e string) string {
-	tpl, err := template.New("error").Parse(errorPage)
+func GetErrorPage(path, name string, e string) string {
```

Comment:
I think it should be should be 
```diff
- func GetErrorPage(path, name string, e string) string {
+ func GetErrorPage(template_path string, name string, e string) string {
```

Reply:
Oh whoops, yeah, I must've managed to undo a rename partially just before I pushed that.